src/styles/blocks/assortiment.scss

```diff
@@ -0,0 +1,113 @@
+.comparing {
+  color: $text-white;
+
+  &__photo {
+    width: 100%;
+    height: 100%;
+    display: block;
+    transition: transform 0.5s ease;
+
+    &:hover {
+      transform: scale(120%);
```

Comment:
![opera_eXacEAUW1U](https://github.com/mate-academy/layout_landing-page/assets/104684446/710dd620-98b4-4438-9532-a443ba87de59)

Scale overlaps neighboring elements, this should not happen